src/renderer/src/routes/pages/Wallets/index.tsx

```diff
@@ -41,6 +49,45 @@ export const WalletsPage = () => {
     setIsReordering(false)
   }
 
+  const timeToDate = (unixTime: number): string => {
+    const date = new Date(unixTime * 1000) // Converte para milissegundos
+    return date.toLocaleDateString() // Use toLocaleDateString() para obter a representação da data
+  }
+
+  const timeToHour = (unixTime: number): string => {
+    const date = new Date(unixTime * 1000) // Converte para milissegundos
+    return date.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' }) // Use toLocaleTimeString() para obter a representação da hora
+  }
+
+  const handleCopyHash = async (hash: string) => {
+    await navigator.clipboard.writeText(hash)
+  }
+
+  const hashToContractName = (address: string, contractHash: string): string => {
+    // const contract = await bsAggregator.getBlockchainByAddress(address)?.blockchainDataService.getContract(contractHash)
+    // return contract?.name ?? ''
+    return address === contractHash ? 'BurgerNEO' : 'GasToken'
+  }
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const allAccounts = accounts.filter(account => account.idWallet === selectedWallet?.id)
+
+      if (allAccounts.length > 0) {
+        const response = await bsAggregator.blockchainServicesByName[
+          allAccounts[0].blockchain
+        ].blockchainDataService.getTransactionsByAddress({
+          address: allAccounts[0].address,
+          page: 1,
+        })
+
+        setTransactions(response.transactions)
+      }
+    }
+
+    fetchData()
+  }, [accounts, bsAggregator.blockchainServicesByName, selectedWallet]) // Add other dependencies as needed
```

Comment:
Use react-query for that, it improves performance.

Reply:
This part will be done in another issue, as discussed earlier. It's something more complex, and for now, only the design structure has been created.

Reply:
Sorry. I had forgotten about that.